### ISSUE TITLE
[Accessibility] Add alt param to `capturePlot()`

### ISSUE THREAD

**Issue by jooyoungseo** [ramnathv/htmlwidgets#438]
Unlike `plotTag()`, `capturePlot()` does not have an `alt` param, which is a critical component for accessibility.

Please add this with default value as "Plot object" so that any NULL-alt plots can be also recognized by screen readers.

If possible, please add a condition to check whether current graph is ggplot2 object having alt attribute so that `g$labels$alt` can be passed to alt param when available.

**Comment by jooyoungseo** [comment]
Moving to htmltools.
